apicast/src/apicast.lua

```diff
@@ -30,6 +31,41 @@ function _M.init()
   end
 end
 
+function _M.init_worker()
+  local interval = tonumber(os.getenv('AUTO_UPDATE_INTERVAL'), 10)
```

Comment:
For example in javascript it does not work like that. So honestly I don't know what to expect :)